playbooks/roles/capsule/tasks/devel_install.yml

```diff
@@ -0,0 +1,19 @@
+- name: 'start rails server'
+  command: /home/vagrant/.rvm/bin/rvm ruby-2.2.4 do rails s
+  args:
+    chdir: /home/vagrant/foreman
+  delegate_to: "{{ capsule_server }}"
+  async: 1000
+  poll: 0
+  become: yes
+  become_user: vagrant
+
+- name: 'Install foreman-installer-katello'
+  yum: name="foreman-installer-katello"
+  delegate_to: "{{ capsule_server }}"
+
+- name: 'Add group foreman'
```

Comment:
Another thing that is required for some reason when installing a capsule in a dev environment